File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/ThanhVienChucNang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThanhVienChucNang } from 'src/app/shared/ThanhVienChucNang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThanhVienChucNangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThanhVienChucNang";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/ThanhVienChucNangService.cs
namespace Service.Implement
{
    public class ThanhVienChucNangService : BaseService<ThanhVienChucNang, IThanhVienChucNangRepository>
    , IThanhVienChucNangService
    {
    private readonly IThanhVienChucNangRepository _ThanhVienChucNangRepository;
    public ThanhVienChucNangService(IThanhVienChucNangRepository ThanhVienChucNangRepository) : base(ThanhVienChucNangRepository)
    {
    _ThanhVienChucNangRepository = ThanhVienChucNangRepository;
    }
    }
    }

